src/client/containers/OrderConfirmationPage/OrderConfirmationPage.styles.css

```diff
@@ -1,3 +1,57 @@
-div {
-  color: #687808;
+.confirmation-page {
+    align-items: center;
+    margin: 4%;
 }
+
+.confirmation-page .inner-div {
+    background-color: #F6F8EB;
+}
+
+.confirmation-page .wrapper2 {
```

Comment:
Here is another hint, the fact that `wrapper` and `wrapper2` have no overlap in the code means they likely should not have any overlap in their names! 

So `wrapper` on looks like some kind of `.thank-you-box` and `wrapper2` looks like a `follow-up-actions-box`.

Reply:
Thank you .. Changed